Source/Scene/Scene.js

```diff
@@ -2562,6 +2562,9 @@ define([
 
         us.update(frameState);
 
+        var backgroundColor = defaultValue(scene.backgroundColor, Color.BLACK);
+        us.updateBackgroundColor(backgroundColor);
```

Comment:
Still store in `UniformState`, and update in `UniformState.update`. And in `Scene` set `frameState.backgroundColor` in `updateFrameState`.